### ISSUE TITLE
LinkML home page may be too YAML centric? Try to include CSV, JSON models in our Tutorial in order to encourage a diverse community

### ISSUE THREAD

**Issue by hsolbrig** [linkml/linkml#1036]
While it makes sense to use YAML heavily in the LinkML introductory material, I wonder whether we might want to reviese the [documentation home page](https://linkml.io/) to make it clear that LinkML models can be authored in YAML, JSON, CSV(?) or created dynamically from other inputs? 

It might be worthwhile to document any model importers that exist today as well as to mention that others are (or could be) in the pike if needed.  In particular, I'd like to see a `JSON-Schema`, `OWL`, (whatever Google's modeling language is), and, someday, `FHIR` and `UML`.

**Comment by Shalsh23** [comment]
I am a linkml beginner exploring the how to validate csv data files for one of our projects. As a user, I would appreciate addition of some more information regarding certain aspects of the linkml-csv workflow. More specifically, an example showing how to validate csv data files using linkml schema. The additional parameter requirements such as `index_slot` are mentioned in the [CSVs and Tabular Data](https://linkml.io/linkml/data/csvs.html) section but can be a little hard to follow for someone new. I would be happy to collaborate on a PR for such an example as I work through this myself.

**Comment by cmungall** [comment]
Thanks for the input @Shalsh23!

Yes it can be hard figuring out where to dive in if your main use case is validating CSVs. The concept of containers is explained in https://linkml.io/linkml/intro/tutorial02.html but this may seem a bit abstract if all you want to do is validate CSVs.

A PR would be most welcome! Were you thinking of adding to the tutorial, or more examples on csvs.html, or a new howto guide?
